static/js/apps/eval_retrieval_generation/app.tsx

```diff
@@ -115,6 +127,9 @@ export function App(props: AppPropType): JSX.Element {
             Google Sheet Link
           </a>
           <p>Signed in as {user.email}</p>
+          {!(allQuery && allCall && doc && sessionQueryId && evalType) && (
```

Comment:
why only add loading explanation for the query loading .. maybe instead, we can have:
- `const initialLoadCompleted = allQuery && allCall && doc && sessionQueryId && evalType`
- if `!initialLoadCompleted`, just show "loading ..."
- if `initialLoadCompleted`, show the app

& same thing for the sxs app

Reply:
Not sure what you meant about only adding for query loading, but I did consolidate into `initialLoadCompleted`. For the sxs app everything is already consolidated into `docInfo` so didn't change anything.